src/plane.ts

```diff
@@ -0,0 +1,38 @@
+// todo:
+// - pitch ctrl derivs
+
+import * as mathjs from 'mathjs';
+import * as THREE from 'three';
+
+export class Plane {
+  wingspan: number;
+  length: number;
+  chord: number;
+  model: THREE.Group;
+
+  //flightTensor: [[number]];
+  constructor() {
+    //this.flightTensor = [
+    //]
+    this.wingspan = 0.9;
+    this.length = 0.6;
+    this.chord = 0.08;
+    const boxGeo = new THREE.BoxGeometry(
+      this.wingspan,
+      this.length,
+      this.length * this.chord
+    );
+    const coneGeo = new THREE.ConeGeometry(
+      this.wingspan * 0.1,
+      this.chord * 1.1,
+      50,
+      16
```

Comment:
Doing it this way serves as clear documentation for what these values are. I'm less inclined to suggest it for the multipliers applied to wingspan & chord directly above because those seem like fudged numbers to make things look right, where "50" and "16" seem much less arbitrary.

But if 0.1 and 1.1 are some aviation constant they should be turned into class constants

Reply:
All four numbers are fudged to get something that looks flatish like a wing and another thing that points in a "forward". The intent is to eventually allow more reasonable definition of what the plane looks like, but that comes after getting some actual kind of simulation going. I changed "16" to "1", as number of height segments doesn't seem to have a practical effect other than making more polygons to be rendered.

Reply:
I'm considering marking this "resolved" for now as implementing more sophisticated geometry generation is a few bullet points down in my development plan

Reply:
SGTM.

At work prototypes only get checked into an experimental area that doesn't require code reviews. When developing something iteratively over the course of several pull requests, we tend to package things into groups of "good code that doesn't do anything" rather than "prototype code that works." That way we don't have to remember to clean up anything later -- oftentimes you never get a chance to come back and clean things up later.

This isn't exactly useful advice to you right now because our goal is to look at your code and not to build something that will last. But hopefully it'll explain some of my inevitable future nitpicks.

Reply:
That makes sense. The isolation of prototype code sounds great. For now I'm isolating the standin "geometry" generation to a method so at least all it'll take is calling a different geometry generation method in the constructor.